scripts/docs.sh

```diff
@@ -11,4 +11,5 @@ python3.7 scripts/docs.py
 python3.7 -c 'import os, pkg_resources; [e.load() for e in pkg_resources.iter_entry_points("console_scripts") if e.name.startswith("sphinx-build")][0]()' -b html docs pages \
   || (echo "[ERROR] Failed run sphinx, is it installed (pip install -U .[dev])?" 1>&2 ; exit 1)
 cp -r docs/images pages/
+wget -P pages/_static "https://raw.githubusercontent.com/python/python-docs-theme/master/python_docs_theme/static/copybutton.js"
```

Comment:
Nice!!! Can we pin this to a hash and using `sha384sum` to verify the contents are what we expect? The [MNIST](https://intel.github.io/dffml/usage/mnist.html) demo has an example of how to verify `sha384sum`s. [dffml.util.net.cached_download](https://intel.github.io/dffml/api/util/net.html#dffml.util.net.cached_download) has an example of how to calculate the hash you want to use for comparison.